Dockerfile.ruff

```diff
@@ -0,0 +1,13 @@
+FROM python:3.11-slim
+
+# Install ruff
+RUN pip install ruff
+
+# Set working directory
+WORKDIR /app
+
+# Copy source code
+COPY . .
```

Comment:
quick question, should we ADD instead of COPY so when we ask ruff to fix code style it will affect the original code not only internal copy of the code?